desktop/src/main/java/bisq/desktop/main/account/content/seedwords/SeedWordsView.java

```diff
@@ -163,7 +163,18 @@ public void activate() {
         seedWordsTextArea.getStyleClass().remove("validation-error");
         restoreDatePicker.getStyleClass().remove("validation-error");
 
+        new Popup().warning(Res.get("account.seed.backup.warning"))
+                .onAction(() -> {
+                    activatePartTwo();
```

Comment:
Maybe you could make this more verbose. e.g.
```suggestion
                    showSeedPhrase();
```